static/main.js

```diff
@@ -19,31 +19,68 @@ function bracketDevicePixelRatio() {
     return brackets[brackets.length - 1];
 }
 
+/**
+ * Finishes setting up the map
+ *
+ * @param {Object|null} config Config object
+ * @param string [config.attribution] Attribution text to show in footer; see below for default
+ * @param number [config.maxzoom=18] Maximum zoom level
+ */
+function setupMap( config ) {
+	var layerSettings,
+		query = '',
+		matchLang = location.search.match(/lang=([-_a-zA-Z]+)/);
+
+	if (matchLang) {
+		query = '?lang=' + matchLang[1];
+	}
+
+	config = config || {};
+
+	layerSettings = {
+		maxZoom: config.maxzoom ? config.maxzoom : 18,
+
+		// TODO: This is UI text, and needs to be translatable.
+		attribution: config.attribution ? config.attribution : 'Map data &copy; <a href="http://openstreetmap.org/copyright">OpenStreetMap contributors</a>',
+
+		id: 'map-01'
+	};
+
+	// Add a map layer
+	L.tileLayer(style + '/{z}/{x}/{y}' + scalex + '.png' + query, layerSettings).addTo(map);
+
+	// Add a km/miles scale
+	L.control.scale().addTo(map);
+
+	// Update the zoom level label
+	map.on('zoomend', function () {
+		document.getElementById('zoom-level').innerHTML = 'Zoom Level: ' + map.getZoom();
+	});
+
+	// Add current location to URL hash
+	var hash = new L.Hash(map);
```

Comment:
Put all 'var xxx' up top

Reply:
This was already like that.  However, I should move it like the others.

Reply:
Actually, it's never used.